tools/wptrunner/wptrunner/browsers/firefox.py

```diff
@@ -93,7 +93,7 @@ def executor_kwargs(test_type, server_config, cache_manager, run_info_data,
         executor_kwargs["reftest_internal"] = kwargs["reftest_internal"]
         executor_kwargs["reftest_screenshot"] = kwargs["reftest_screenshot"]
     if test_type == "wdspec":
-        fxOptions = {}
+        fxOptions = {}  # noqa: N806
```

Comment:
I thought we'd want to stay consistent with the capability name?

Reply:
@Ms2ger any opinion on naming lints? @jgraham is complaining because of benefit/reward.

Reply:
Specifically I claim that the `N*` lints can't find bugs by definition, and also aren't clever enough to work out when the style rules are being violated intentionally (e.g. when giving a class used as a decorator or other callable a lowercase name).

From the patch it looks like we have a number of exceptions to these lints, resulting in ugly `noqa` comments, but they don't result in any other substantial changes. Future exceptions will probably waste author's time since they're unlikely to remember the random 3 digit code for the lint, so they're going to end up noticing when travis fails.

I don't mind fixing specific instances of these rules being violated (e.g. this one), but I don't think people violating these naming conditions is enough of an issue to warrant the downsides of enabling the lints.

Reply:
Just rename them.